plugin/SearchBarItem.qml

```diff
@@ -162,11 +184,14 @@ BackgroundItem {
         EnterKey.onClicked: {
             if (text != "") {
                 _searchText = text
+                focus = false // Need to be called after saving text
+                              // but before the request, see the code
+                              // in onActiveFocusChanged, when activeFocus is false.
```

Comment:
Not sure I'm still following. onActiveFocusChanged clears text in some cases, so by this requestSearch("") gets emitted? Maybe related but now searching for bogus, "asdf", every other time when I focus the editor and click enter it either searches or just dismisses. Would work better by just leaving focus where it was and emitting _searchText with requestSearch?

Nitpick: suggest having multiline comments before the line, otherwise automatic indentation mangles this at some point.